src/GraphQl/Action/GraphQlPlaygroundAction.php

```diff
@@ -11,7 +11,7 @@
 
 declare(strict_types=1);
 
-namespace ApiPlatform\Core\GraphQl\Action;
+namespace ApiPlatform\GraphQl\Action;
```

Comment:
Why are you changing the namespace?

Reply:
Oh, sorry. Must have been some hickup in my IDE. I'll reverse that